db_migrations/0015_recreate_media_files.sql

```diff
@@ -0,0 +1,44 @@
+-- Migration 0015: Recreate media_files table with clean schema for media cache
+--
+-- This migration drops the experimental media_files table from migration 0002
+-- and recreates it with a proper schema designed for:
+-- 1. Group profile images (MIP-01)
+-- 2. Future group media files (MIP-04)
+--
+-- Key features:
+-- - Cross-account cache with reference tracking
+-- - Normalized columns (no JSONB)
+-- - Hash-based storage organization
+-- - LRU eviction support via accessed_at
+
+-- Drop old experimental media_files table
+DROP TABLE IF EXISTS media_files;
+
+-- Create new media_files table with clean schema
+CREATE TABLE media_files (
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    mls_group_id BLOB NOT NULL,
+    account_pubkey TEXT NOT NULL,
+    file_path TEXT NOT NULL,
+    file_hash TEXT NOT NULL,          -- Hex-encoded hash of ENCRYPTED data (image_hash for group images, per MIP-01)
+    mime_type TEXT NOT NULL,          -- Canonical MIME type (e.g., "image/jpeg")
+    media_type TEXT NOT NULL,         -- "group_image" or "chat_media"
+    blossom_url TEXT,                 -- Optional: NULL for default server, set for custom
+    nostr_key TEXT,                   -- Optional: For future MIP-04 cleanup
+    dimensions TEXT,                  -- Optional: Image/video dimensions as "widthxheight" (e.g., "1920x1080")
+    blurhash TEXT,                    -- Optional: BlurHash string for progressive loading
+    created_at INTEGER NOT NULL,
+    accessed_at INTEGER NOT NULL,     -- For LRU eviction
```

Comment:
This seems a bit premature but ok, it doesn't hurt too much

Reply:
Actually thinking about it a bit more, I think we should remove this. We might want to implement this in a different way with a different table maybe in the future. I just don't think we need to really think about this too much and it's gonna complicate things to keep it 'in sync' (update in the db when we grab the file)